docs/source/gbd2017_models/causes/neoplasms/colon_and_rectum_cancer/cancer_model.rst

```diff
@@ -1,244 +1,274 @@
-   Disease Study 2017. Lancet 2018; 392: 1789–858`
-   (pp. 310-317)
+.. _2017_cancer_model_colon_and_rectum_cancer:
+
+=======================
+Colon and Rectum Cancer
+=======================
+
+.. contents::
+  :local:
+
+Disease Overview
+++++++++++++++++
+
+.. todo::
+
+ add CRC overview
+
+.. list-table:: Malignant neoplasm of colon and rectum cancer 
+   :widths: 10 10
+   :header-rows: 1
+
+   * - ICD 10
+     - ICD 9
+   * - C18-C19.0, C20, C21-C21.8, Z12.1-Z12.13 (Z*=screening)
+     - 153-154.9, V76.41, V76.5-V76.52 (V*=screening)
+
+
+GBD 2017 Modeling Strategy
+++++++++++++++++++++++++++
+
+.. todo::
+
+ add GBD CRC modeling strategy
+
+
+Cause Hierarchy
+------------------------------------------
+       
+.. image:: colon_and_rectum_cancer_cause_hierarchy.svg
+
+
+Restrictions
+------------
+
+The following table describes any restrictions on the effects of this cause 
+(such as being only fatal or only nonfatal), as well as restrictions on the 
+age and sex of simulants to which different aspects of the cause model applies.
+
+.. list-table:: Restrictions
+   :widths: 10 10 10
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLL age group end
+     - 95 plus
+     - GBD age group id 235
+   * - YLD age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLD age group end
+     - 95 plus
+     - GBD age group id 235
+
+
+Vivarium Modeling Strategy
+++++++++++++++++++++++++++
+
+Things to consider: 
+
+1. Duration of remission from clinical CRC *over 10 years*, to be confirmed 
+   with GBD modeler.
+
+.. todo::
+
+   Add assumptions and limitations.
+
+
+Compartmental Diagram
++++++++++++++++++++++
+
+  .. image:: colon_and_rectum_cancer_cause_model_diagram.svg
+
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 5 10 20
+   :header-rows: 1
+   
+   * - State
+     - State name
+     - Definition
+   * - S
+     - Susceptible 
+     - Healthy OR with asymptomatic condition, but not screen-detectable
+   * - PC
+     - Pre-clinical screen-detectable colon and rectum cancer
+     - with asymptomatic condition and screen-detectable
+   * - C
+     - Clinical colon and rectum cancer
+     - With symptomatic condition
+   * - R
+     - Recovered
+     - Recovered from clinical colon and rectum cancer, not susceptible
+
+.. list-table:: State Data
+   :widths: 5 5 20 20
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - (1 - prev_PC - prev_C)
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     - 
+   * - PC
+     - prevalence
+     - prev_c441 * MST / AST

```

Comment:
I think AST is not needed here

Reply:
Are you prefer the prevalence of PC equals to `i_PC * MST` if we want it be consistent with other measures?

Reply:
prev_c441 * MST / AST is consistent with the approach we took for lung cancer (scale prevalence to the relative duration of preclinical and clinical states)